Add unlimited plugin

nilic opened the pull request:
A simple plugin which shows information about running containers with no CPU and/or memory limits set in order to detect potential noisy neighbors and optimize cluster's performance and resource consumption.

Analyzed containers can be filtered on a namespace or pod label basis, and output can be displayed in table, JSON and YAML formats.

ahmetb commented:
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

nilic commented:
Thanks for the comments. Since this is my first plugin, I was looking into developing something simple to do one thing and do it well :)

I have added [auth plugins import](https://github.com/nilic/kubectl-unlimited/commit/7eee95920af1dd9691244166064bceeda8fbff41), built a new release and updated submitted krew manifest.

ahmetb commented:
/lgtm
/approve
Thanks!

Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.
